coalib/output/Logging.py

```diff
@@ -4,25 +4,48 @@
 import logging
 import logging.config
 
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
 
-def configure_logging():
+
+def configure_logging(new_log_level=logging.INFO, incremental=False):
     """
     Configures the logging with hard coded dictionary.
+
+    If the incremental param is ``True``, all the log-handlers configuration
+    other than level will be ignored.
+    Thus, use it only when changing the log_level.
+    When changing ``log_level``, ``json-handler``, ``console-handler``,
+    and ``file-handler`` will have to exist.
+    ``json-handler`` is created as a null handler to avoid this error.
+
+    :param new_log_level:
```

Comment:
I would just call the param `log_level`. When using it with `incremental=False`, this "new_" doesn't make so much sense imo ;)